src/test/java/hydrovor/HydrovorTest.java

```diff
@@ -1,5 +1,77 @@
 package hydrovor;
 
+import junit.framework.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mockito;
+
+import static org.mockito.Mockito.when;
+
 public class HydrovorTest {
 
+    private Tank tank= Mockito.mock(Tank.class);
```

Comment:
Do not initialize mocks during object construction as it may be reused between tests.
Use `@Before` method.